### ISSUE TITLE
keys input don't work

### ISSUE THREAD

**Issue by N4RR4D0R** [korlibs/korge#12]
Any function about keys don't work. High-Level and Low-Level APIs.
I use Kotlin 1.13.20 and Korge 1.0.3

**Comment by soywiz** [comment]
Which target? JVM, JS, Android, iOS, Linux, Windows or macOS?

Relevant project: https://github.com/korlibs/korui/tree/master/korgw/src

**Comment by N4RR4D0R** [comment]
Keys API not work on X11 and Wayland.
Mouse API only works on low-level, High-level not work

**Comment by N4RR4D0R** [comment]
I tried on Vivaldi too,It doesn't work

**Comment by N4RR4D0R** [comment]
My test code is this
`import com.soywiz.klock.milliseconds
import com.soywiz.korev.*
import com.soywiz.korge.*
import com.soywiz.korge.input.*
import com.soywiz.korge.scene.Module
import com.soywiz.korge.scene.Scene
import com.soywiz.korge.view.*
import com.soywiz.korinject.AsyncInjector
import com.soywiz.korio.async.delay
import com.soywiz.korio.async.launchImmediately

suspend fun main() = Korge(Korge.Config(module = MyModule))

object MyModule:Module(){
    override val mainScene=GameScene::class

    override suspend fun init(injector: AsyncInjector):Unit =injector.run{
        mapPrototype{GameScene()}
    }
}

class GameScene:Scene(){
    var count=0
    override suspend fun Container.sceneInit() {
        addEventListener<Event> {
            println("Event")
        }
        addEventListener<MouseEvent> {
            if(it.type==MouseEvent.Type.CLICK){
                count++
            }
        }
        onKeyDown {
            count++
        }
        keys.down(Key.A){
            println("A")
        }
        text("Click $count", 100.0){
            onClick {
                count++
                println("Hola")
            }
            keys.down(Key.A){
                println("A")
            }
            position(100,100)
            launchImmediately {
                while (true) {
                    this.text = "Click $count"
                    position(100,100)
                    delay(1.milliseconds)
                }
            }
        }
    }
}`

**Comment by soywiz** [comment]
Yeah. Confirmed. I'm looking at it. In the last weeks I changed `korui` by `korgw` and it seems I missed the keys part.

I have created a sample here to verify it works once I have implemented the fix:
https://github.com/korlibs/korge-samples/commit/621b9af849cb0b6d375402743d09843b899eabc6

**Comment by lbrasseur** [comment]
Input keyboard seems to be failing again. I tried sample-input with korgePluginVersion=1.5.6.0 and KeyUp event is not triggered on JVM, but KeyDown is ok. On native Linux, the opposite happens (KeyUp works, KeyDown does not). On JS both work ok.
